src/datafactory/azext_datafactory/generated/custom.py

```diff
@@ -553,8 +581,10 @@ def datafactory_pipeline_update(instance,
         instance.annotations = annotations
     if run_dimensions is not None:
         instance.run_dimensions = run_dimensions
+    if duration is not None:
+        instance.elapsed_time_metric.duration = duration
     if folder_name is not None:
-        instance.name_folder_name = folder_name
+        instance.folder.name = folder_name
```

Comment:
Is this folder_name  flattened from folder.name ? I wonder if this instance path is correct ?

Reply:
Yes. I think it's correct.